test/react/hooks/use-room-reactions.test.tsx

```diff
@@ -7,6 +7,7 @@ import { makeTestLogger } from '../../helper/logger.ts';
 import { makeRandomRoom } from '../../helper/room.ts';
 
 let mockRoom: Room;
+let mockLogger: ReturnType<typeof makeTestLogger>;
```

Comment:
Not sure how this works. Is this supposed to have a value?

Reply:
The value is set in the `beforeEach` function, so will always be set before each test.